Resources/views/backend/swag_backend_order/controller/main.js

```diff
@@ -617,6 +620,12 @@ Ext.define('Shopware.apps.SwagBackendOrder.controller.Main', {
 
                 me.validationMail = pluginConfigObj.data.validationMail;
                 me.desktopTypes = pluginConfigObj.data.desktopTypes;
+                me.sendMail = pluginConfigObj.data.sendMail;
+                if (me.sendMail == 1) {
+                    me.orderModel.set('sendMail', 1);
+                    //Set checked class for table
+                    document.getElementById('sendMailChk').classList.add('x-form-cb-checked');
```

Comment:
At the moment I have no idea what you mean exactly, because normally I do nothing in the backend :) But I´ll try to reach it out and comment again. Thanks for your help.